packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -197,11 +432,69 @@ export class Table extends Resource {
       actions: ['dynamodb:ListStreams'],
       resourceArns: ['*'],
     });
- }
+  }
+
+  /**
+   * Creates a Table construct that represents an external table.
+   *
+   * @param scope The parent creating construct (usually `this`).
+   * @param id The construct's name.
+   * @param attrs A `TableAttributes` object.
+   */
+  public static fromTableAttributes(scope: Construct, id: string, attrs: TableAttributes): ITable {
```

Comment:
We also need 2 more methods: `fromTableName(scope: Construct, id: string, tableName: string): ITable` and `fromTableArn(scope: Construct, id: string, tableArn: string): ITable`.